janusgraph-dist/janusgraph-dist-hadoop-2/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openjdk:8
+
+ENV PATH "$JAVA_HOME/bin:$PATH"
+RUN apt-get update -y
+RUN apt-get install -y zip
```

Comment:
@analytically am I allowed to combine an ENV directive with a RUN directive? I understand that I can write `RUN apt-get update -y && apt-get install -y zip` and `RUN unzip -q ${server_base}.zip && ln -s ./${server_base} ./janusgraph`